ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityInstanceSelectAnswer.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map, take } from 'rxjs/operators';
+
+import { ActivityInstanceSelectOptionDto } from '../../models/activity/activityInstanceSelectOptionDto';
+import { UserServiceAgent } from './userServiceAgent.service';
+
+interface GetOptionsResponse {
+  results: ActivityInstanceSelectOptionDto[];
+}
+
+@Injectable()
+export class ActivityInstanceSelectAnswerService extends UserServiceAgent<GetOptionsResponse> {
+  getOptions(questionStableId: string): Observable<GetOptionsResponse['results']> {
```

Comment:
I wondering why is not just 
`Observable<ActivityInstanceSelectOptionDto[]>` ?

Reply:
I've used `GetOptionsResponse['results']` so that if any changes are made to `GetOptionsResponse` interface we wouldn't need to tweak the return type of `getOptions` method.